app/public/js/init.js

```diff
@@ -2,3 +2,4 @@
 require('./cookieMessage');
 require('./geoLocation');
 require('./menuToggle');
+require('./c2sScripts');
```

Comment:
Can this script be renamed to be less generic? Something like `toggleResults` would make it easier to know what is does.